var/spack/repos/builtin/packages/libedit/package.py

```diff
@@ -30,7 +30,6 @@ class Libedit(AutotoolsPackage):
     homepage = "http://thrysoee.dk/editline/"
     url      = "http://thrysoee.dk/editline/libedit-20150325-3.1.tar.gz"
 
-    version('3.1', '43cdb5df3061d78b5e9d59109871b4f6',
-            url="http://thrysoee.dk/editline/libedit-20150325-3.1.tar.gz")
+    version('20150325-3.1', '43cdb5df3061d78b5e9d59109871b4f6')
```

Comment:
i think this is the only difficult one: both [Ubuntu](https://launchpad.net/ubuntu/+source/libedit) and [Debian](https://packages.qa.debian.org/libe/libedit.html) name versions as `3.1-<date>`. For example, the current release on their site is http://thrysoee.dk/editline/libedit-20160903-3.1.tar.gz (note `2016`). I would add here `ulr_for_versions` to make it into `3.1-20150325` just to be consistent with LInux distros. Maybe at some point some package will require `3.2:` or `:3.0`, who knows how it is used. For example, in Ubuntu you see that there was `2.11-20080614`.

p.s. from the webpage one can't really tell how versions should be used, but release history on Linuxes help here. I also downloaded the archived, but could not really find how versions are mentioned.